ui/apps/platform/cypress/integration/risk/risk.test.js

```diff
@@ -32,31 +32,20 @@ describe('Risk', () => {
             cy.get('.rt-th:contains("Priority")');
         });
 
-        it('should open side panel for deployment', () => {
+        it('should open detail page for deployment', () => {
             visitRiskDeployments();
             viewRiskDeploymentByName('collector');
         });
 
         // TODO add relevant tests for error messages in PatternFly
 
-        it('should open the panel to view risk indicators', () => {
+        it('should open the detail page to view risk indicators, deployment details, and process discovery tabs', () => {
```

Comment:
Since we are only asserting on the presence of the tabs currently, IMO this doesn't need to be multiple tests.